### PR TITLE
Sankey: properly set coordinates of node's origin prior to drag operation

### LINKED ISSUE #3563: Sankey nodes jump on dragging
When you drag a sankey node, on first mouse move it jumps vertically (or horizontally if the diagram is horizontal). Bug exists on master but not in the latest release 1.44.4
sankey-drag-jump
cc @antoinerg

### PR DESCRIPTION
**Pull request by antoinerg**
Fixes #3563

### CODE REVIEW

**Comment by alexcjohnson** [comment]
Yep, that fixes it for me! Can we 🔒 it with a test?

**Comment by antoinerg** [comment]
I added a new test in 1f73366 for drag interactions. The test fails without the fix in introduced in08e5a50!

Thank you @alexcjohnson for catching this :beetle: !

File: test/jasmine/tests/sankey_test.js [hunk 1]
```diff
@@ -905,6 +906,57 @@ describe('sankey tests', function() {
         });
     });
 
+    describe('Test drag interactions:', function() {
```
Comment (etpinard) [inline test/jasmine/tests/sankey_test.js]: Wow! Fantastic test! Thanks!

**Comment by etpinard** [comment]
Nicely done :dancer:

**Pull request merged**